File: Programa/Codigo/DadosAPI/olimpiclink.database/olimpiclink.database/Models/Events/PostEventModel.cs
namespace olimpiclink.database.Models.Events
{
    public record PostEventModel
        (
            int place_id,
            int comunity_id,
            int leader_id,
            string nameEvent,
            string descriptionEvent,
            string dateTimeEvent,
            string closingDateTimeEvent,
            string[]? pictures_event,
            string endereco
        )
    {}
}

File: Programa/Codigo/app/src/main/java/devsystem/olimpiclink/model/PostEventModel.kt
package devsystem.olimpiclink.model

import java.util.Date

data class PostEventModel(
    val place_id: Int,
    val comunity_id: Int,
    val leader_id: Int,
    val nameEvent: String,
    val descriptionEvent: String,
    val dateTimeEvent: String,
    val closingDateTimeEvent: String,
    val pictures_event: Array<String>?,
    val endereco : String
)

